srcpkgs/ClusterSSH/template

```diff
@@ -1,19 +1,26 @@
-# Template build file for 'ClusterSSH'.
+# Template file for 'ClusterSSH'
 pkgname=ClusterSSH
-version=4.13
+version=4.13.2.01
 revision=1
+_distver="${version%.*}_${version##*.}"
 noarch=yes
-wrksrc="App-${pkgname}-${version}"
+wrksrc="clusterssh-${_distver}"
 build_style=perl-ModuleBuild
 hostmakedepends="xterm openssh
  perl-CPAN-Changes perl-Module-Build perl-Try-Tiny perl-Tk
  perl-Exception-Class perl-X11-Protocol perl-X11-Protocol-Other
  perl-File-Slurp perl-File-Which  perl-Readonly perl-Test-Differences
  perl-Test-DistManifest  perl-Test-Pod perl-Test-Pod-Coverage perl-Test-Trap"
 depends="xterm openssh perl-Try-Tiny perl-Tk perl-X11-Protocol perl-X11-Protocol-Other perl-Exception-Class"
-short_desc='A tool for making the same change on multiple servers at the same time'
+checkdepends="perl-Sort-Naturally"
+short_desc="Tool for making the same change on multiple servers at the same time"
 maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
-homepage="http://clusterssh.sourceforge.net"
-license="GPL-1, Artistic"
-distfiles="${SOURCEFORGE_SITE}/clusterssh/App-${pkgname}-${version}.tar.gz"
-checksum=eb0df729340054a787429cfcefac79d28cf9013553d01d079ffd8e82fbf35681
+license="Artistic-1.0-Perl, GPL-1.0-or-later"
+homepage="https://github.com/duncs/clusterssh"
+distfiles="https://github.com/duncs/clusterssh/archive/v${_distver}.tar.gz"
+checksum=be6da42d448d925485d24c060f93215980e4e603476d61e18e79e219facea65a
+nocross=yes
```

Comment:
why? can't this be fixed?

Reply:
It's noarch, nocross is really problem?

Reply:
yes, with our current build system, noarch packages need to be built for every architecture to be added to the repodata